### PR TITLE
OSDOCS#3977: Tech preview of cgroup v2 and crun Release Note

### PR DESCRIPTION
**Pull request by mburke5678**
https://issues.redhat.com/browse/OSDOCS-3977
https://issues.redhat.com/browse/OSDOCS-3978

Previews
[cgroup and crun release notes](http://file.rdu.redhat.com/~mburke/nodes-cgroups-vs-tech-preview-rn/release_notes/ocp-4-12-release-notes.html#ocp-4-12-node-cgroups-v2)
[Technology Preview features, final rows in table](http://file.rdu.redhat.com/~mburke/nodes-cgroups-vs-tech-preview-rn/release_notes/ocp-4-12-release-notes.html#ocp-4-12-technology-preview)

PR to update the rest of the docs: https://github.com/openshift/openshift-docs/pull/49472

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -98,6 +98,11 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 [id="ocp-4-12-nodes"]
 === Nodes
 
+[id="ocp-4-12-node-cgroups-v2"]
+==== Linux control groups version 2 promoted to Techology Preview
+
+{product-title} support for link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[Linux control groups version 2] (cgroups v2) has been promoted to Technology preview. Cgroups v2 is the next version of the kernel link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/resource_management_guide/ch01[control groups] and offers multiple improvements.
```
Comment (pneedle-rh) [inline release_notes/ocp-4-12-release-notes.adoc:104]: You could update "Technology preview" to "Technology Preview".

Comment (pneedle-rh) [inline release_notes/ocp-4-12-release-notes.adoc:104]: From a review of the OCP docs library, I see that cgroup is stated in several ways:

* cgroup
* cgroups
* Cgroup
* Cgroups
* CGroup
* CGroups

In https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html, the following is mentioned:

> “cgroup” stands for “control group” and is never capitalized.

I also noticed singular or plural use of the term in that document. 

Given this, you could consider changing "Cgroups" to "cgroups" and using the lowercase version of the term throughout the PR.

Comment (pneedle-rh) [reply release_notes/ocp-4-12-release-notes.adoc:104]: You could consider updating "and offers multiple improvements" to something like the following examples:

> and v2 offers multiple improvements
> and it offers multiple improvements
> and the update offers multiple improvements

Alternatively, you could split that sentence into two and provide some example high-level improvements. For example:

```
cgroups v2 is the next version of the kernel [control groups](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/resource_management_guide/ch01). cgroups v2 offers multiple improvements, including <improvements>.
```

**Review by pneedle-rh — COMMENTED** [review]
@mburke5678 I have completed my review of this PR. Please let me know if you have any questions about my suggestions. Thanks!

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (mburke5678) [reply release_notes/ocp-4-12-release-notes.adoc:104]: @pneedle-rh Thank you for your review. You are correct about your two typo comments. I did catch those on my final review but failed to push those changes. Sorry to have wasted your time. I am taking your third suggestion, to split the sentence. Sounds much better. Thanks again.

Comment (pneedle-rh) [reply release_notes/ocp-4-12-release-notes.adoc:104]: Thank you Michael. I noticed that for some reason the link in my example uses Markdown format. I mean to use AsciiDoc format:

```
cgroups v2 is the next version of the kernel link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/resource_management_guide/ch01[control groups]. cgroups v2 offers multiple improvements, including <improvements>.
```

**Pull request merged**